bin/device-install.sh

```diff
@@ -138,15 +138,15 @@ if [ -f "${FILENAME}" ] && [ -n "${FILENAME##*"update"*}" ]; then
 
 	# littlefs* offset for BigDB 8mb and OTA OFFSET.
 	for variant in "${BIGDB_8MB[@]}"; do
-		if [ -n "${FILENAME##*"$variant"*}" ]; then
+		if [ ! -n "${FILENAME##*"$variant"*}" ]; then
```

Comment:
I *think* this is equivalent:

```suggestion
		if [ -z "${FILENAME##*"$variant"*}" ]; then
```

Reply:
This will be true if empty string, or if the variable is unset